curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/646477c82475ffaf0c3c0771.md

```diff
@@ -0,0 +1,181 @@
+---
+id: 646477c82475ffaf0c3c0771
+title: Step 83
+challengeType: 0
+dashedName: step-83
+---
+
+# --description--
+
+While asynchronous, or <dfn>async</dfn>, code can be difficult to understand at first, it has many advantages. One of the most important is that it allows you to write non-blocking code.
```

Comment:
`async` is technically a reserved keyword in JavaScript. Do we want to define it as a term here? I worry that might get confusing.